pkg/controller/direct/redis/cluster/mapper.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"time"
+
+	pb "cloud.google.com/go/redis/cluster/apiv1/clusterpb"
+	krm "github.com/GoogleCloudPlatform/k8s-config-connector/apis/redis/v1alpha1"
+	refs "github.com/GoogleCloudPlatform/k8s-config-connector/apis/refs/v1beta1"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func Cluster_CreateTime_FromProto(mapCtx *MapContext, in *timestamppb.Timestamp) *string {
```

Comment:
I think we may be able to automatically generate these (or automatically generate them, then allow overrides) ... but I want to collect a few more examples first